synergy-design-system/synergy-design-system
# feat: ✨ DX: Angular should allow using boolean attributes without square brackets in templates

# Pull Request

## 📖 Description

This PR provides a small change for our angular wrappers, adding the possibility to use short hand notation for `boolean` attributes without angulars... angular syntax :).

### Before:

```html
<syn-input name="something" [required]="true" />
<syn-checkbox name="checkbox" [required]="true" [checked]="true" />
```

### After:

```html
<syn-input name="something" required />
<syn-checkbox name="checkbox" required checked />
```

Both syntaxes are valid from now on, making it possible to use the shorter syntax in the future without breaking current projects relying on the angular-bracket notation.

### 🎫 Issues

Closes #689

## 👩‍💻 Reviewer Notes

Just have a look at the created code for the angular components. Also, play around with the angular demo project to see how this can be mixed and matched to taste.

## 📑 Test Plan

- Check out the repository
- Install and build all
- Move to the angular demo and start it up. This version already uses the short hand syntax where possible
- Change one of the required fields in a demo to the old syntax. Note that there is no difference between both versions.

## ✅ DoD
